dynatrace-aws-logs.sh

```diff
@@ -150,6 +147,65 @@ arguments:
   aws cloudformation describe-stacks --stack-name "$STACK_NAME" --query "Stacks[0].Outputs"
   ;;
 
+"delete")
+
+  function print_help_delete {
```

Comment:
Are we supposed to add this function? 
If we add this, I would expect this does delete stack AND removes all log group subscriptions for this stack. 
Also, this probably needs corresponding description in DT docs

Reply:
We already discussed this in one of our daily meetings.

IMHO the customer needs to have an option to delete the log forwarder. Update the docs YES, but add a note about the need to remove all log group subscriptions for the given stack. Otherwise, we are duplicating code.

Reply:
I think the "delete" option should do this log group subscriptions on its own. 
1) Autodiscover all 2) Pass this to unsubscribe.
This can be easily automated I think and should be from UX perspective

Reply:
I would consider two things here:
1. Is there a scenario, where the end-user does want to delete the log forwarder but keep the logs and subscription groups? If not, then we could add the unsubscribe function.
2. Adding `unsubscribe` as part of `delete` would IMHO make the latter inconsistent with 'deploy' and could confuse the user.

Reply:
1. No, we don't have such scenario 
2. "delete" is almost useless for the user from my point of view. We can name it "purge" or "uninstall" or something else, but I expect full cleanup from this step.
My concern is following: I am a user, you gave me "delete" option, I expect it to delete properly and not leave lots of mess behind. If you want to give me "delete" option which only does like 30% of work then it's better not to give this option

Reply:
`delete` function has been removed and the execution replaced by AWS CLI call.